### DISCUSSION TITLE
精度问题EAO 35 35 Robust 27.2

### DISCUSSION THREAD

**Discussion by Holmes-GU** [STVIR/pysot#440] [Reproduce] [answered]
我用了两张1080ti训练，在照着提供的命令跑了一下代码，在vot2018上最好的epoch是14，acc 59.2 robust 27.2 EAO 35， 与文中的结果差的还是挺大的，不知道是何原因，求指导

**Comment by cunjif** [comment]
可能时卡不够吧

**Comment by ZhiyuanChen** [comment, marked as answer]
与https://github.com/STVIR/pysot/discussions/94 重复
